src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -1087,6 +1091,10 @@ internal override Cci.IMethodReference Translate(MethodSymbol symbol, Diagnostic
             {
                 methodSymbol = AnonymousTypeManager.TranslateAnonymousTypeMethodSymbol(methodSymbol);
             }
+            else if (container.IsTupleType)
+            {
+                container = ((TupleTypeSymbol)container).UnderlyingTupleType;
```

Comment:
And yes, intercepting methods is already removed in the other change. Tuples do not own methods.